spec/features/action_plan_generate_spec.rb

```diff
@@ -8,15 +8,15 @@
 
   let!(:project) do
     FactoryBot.create(:project, parent: parent).tap do |project|
-      FactoryBot.create(:story, title: "Second Story", description: "Second", position: 2, project: project)
-      FactoryBot.create(:story, title: "First Story", description: "First", position: 1, project: project)
+      FactoryBot.create(:story, title: "Second Story", description: "Second", position: 2, project: project, extra_info: "Extra Information")
+      FactoryBot.create(:story, title: "First Story", description: "First", position: 1, project: project, extra_info: "Extra Information")
     end
   end
 
   let!(:project2) do
     FactoryBot.create(:project, parent: parent).tap do |project|
-      FactoryBot.create(:story, title: "Third Story", description: "Third", position: 2, project: project)
-      FactoryBot.create(:story, title: "Forth Story", description: "Forth", position: 1, project: project)
+      FactoryBot.create(:story, title: "Third Story", description: "Third", position: 2, project: project, extra_info: "Extra Information")
+      FactoryBot.create(:story, title: "Forth Story", description: "Forth", position: 1, project: project, extra_info: "Extra Information")
```

Comment:
are these extra_info values needed? the field should be optional, so I think it's better to not add the attribute if not needed, or it will give the impression that it's required for somebody reading the tests

Reply:
I left one of them in along with a comment to point out that the test asserts (by omission) the extra_info content doesn't show up in the action plan